src/commons/api/roas.rs

```diff
@@ -368,43 +555,35 @@ impl fmt::Display for RoaDefinitionUpdates {
     }
 }
 
-impl FromStr for RoaDefinitionUpdates {
+impl FromStr for RoaConfigurationUpdates {
     type Err = AuthorizationFmtError;
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
         let mut added = vec![];
         let mut removed = vec![];
 
         for line in s.lines() {
```

Comment:
So you're removing support for entire comment lines in a block of ROA strings? Is that deliberate? Was that ever officially even supported?